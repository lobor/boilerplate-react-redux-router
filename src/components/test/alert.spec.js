import React from 'react'
import styled from 'styled-components'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import Alert from './../alert'

test('render default alert', () => {
  const tree = renderer.create(<Alert />).toJSON()
  expect(tree).toMatchSnapshot()
})

test('render warning alert', () => {
  const tree = renderer.create(<Alert warning />).toJSON()
  expect(tree).toMatchSnapshot()
})

test('render error alert', () => {
  const tree = renderer.create(<Alert error />).toJSON()
  expect(tree).toMatchSnapshot()
})
