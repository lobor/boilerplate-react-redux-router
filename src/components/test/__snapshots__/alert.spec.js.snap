// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render default alert 1`] = `
.c0 {
  padding: 15px;
  font-size: 14px;
  line-height: 1.5;
  color: #246;
  background-color: #e2eef9;
  border: 1px solid #bac6d3;
  border-radius: 3px;
  margin-bottom: 10px;
}

<div
  className="c0"
/>
`;

exports[`render error alert 1`] = `
.c0 {
  padding: 15px;
  font-size: 14px;
  line-height: 1.5;
  color: #246;
  background-color: #e2eef9;
  border: 1px solid #bac6d3;
  border-radius: 3px;
  margin-bottom: 10px;
  color: #911;
  background-color: #fcdede;
  border-color: #d2b2b2;
}

<div
  className="c0"
/>
`;

exports[`render warning alert 1`] = `
.c0 {
  padding: 15px;
  font-size: 14px;
  line-height: 1.5;
  color: #246;
  background-color: #e2eef9;
  border: 1px solid #bac6d3;
  border-radius: 3px;
  margin-bottom: 10px;
  color: #4c4a42;
  background-color: #fff9ea;
  border-color: #dfd8c2;
}

<div
  className="c0"
/>
`;
